Frends.FTP.ListFiles/Frends.FTP.ListFiles/Definitions/ListObject.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Frends.FTP.ListFiles.Definitions
+{
+    /// <summary>
+    /// Single file data.
+    /// </summary>
+    public class ListObject
+    {
+        /// <summary>
+        /// Last modified timestamp of the file. 
+        /// </summary>
+        /// <example>3.5.2022 0.00.00</example>
+        public DateTime Modified { get; set; }
+
+        /// <summary>
+        /// Filename.
+        /// </summary>
+        /// <example>testfile.txt</example>
+	    public string Name { get; set; }
+
+        /// <summary>
+        /// Full path of the file.
+        /// </summary>
+        /// <example>/testfile.txt</example>
```

Comment:
Lets do an example with a subdirectory to show a slightly more extended example.